### PR TITLE
Snapshot fixes

### PR DESCRIPTION
**Pull request by itsdalmo**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -10,10 +10,12 @@ resource "aws_rds_cluster" "main" {
   master_password              = "${var.password}"
   port                         = "${var.port}"
   engine                       = "${var.engine}"
+  engine_version               = "${var.engine_version}"
   backup_retention_period      = 7
   preferred_backup_window      = "02:00-03:00"
   preferred_maintenance_window = "wed:04:00-wed:04:30"
   snapshot_identifier          = "${var.snapshot_identifier}"
+  final_snapshot_identifier    = "${var.name_prefix}-cluster"
```
Comment (mikael-lindstrom) [inline main.tf]: I was thinking this should only be set if `skip_final_snapshot` is `false` but it looks safe to always set it. I do think the name should be something like `"${var.name_prefix}-cluster-final-snapshot"`

**Review by mikael-lindstrom — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (itsdalmo) [reply main.tf]: I'm worried that the name might be too long if we add "final-snapshot" in there :/

Comment (mikael-lindstrom) [reply main.tf]: Maybe just add final then? Would be nice to have something to easily see that its a final snapshot.

**Review by mikael-lindstrom — APPROVED** [review]
LGTM

**Pull request merged**